test/integration/product-import.spec.coffee

```diff
@@ -122,6 +122,21 @@ sampleCustomerGroup =
 sampleChannel =
   key: 'test-channel'
 
+newState =
+  key: "New"
+  type: "ProductState"
+  initial: true
+
+extantState =
+  key: "AlsoNewButIWantToCheckPopulatingWithANonDefaultState"
```

Comment:
Is this long key necessary? IMHO shorter is better

Reply:
I can certainly move the description from thr key into a comment.